docs/_layouts/default.html

```diff
@@ -38,7 +38,7 @@
 </div>
 
 <footer>
-    <p class="light tiny align_center">Copyright © 2015–2016 Slack Technologies, Inc. and contributors</p>
+    <p class="light tiny align_center">© 2019 Slack Technologies, Inc. and contributors</p>
```

Comment:
no range? i'm cool with that. there was probably some jekyll helper to get the current year, but again, i'm cool with that.